src/context/ColonyContext.tsx

```diff
@@ -12,11 +13,13 @@ interface ColonyContextValue {
   colony?: Colony;
   loading: boolean;
   canInteractWithColony: boolean;
+  error?: ApolloError;
```

Comment:
Do we need that? Doesn't seem to be used anywhere

Reply:
You are right, I'll remove it.